app/views/CreateDiscussionView/SelectChannel.tsx

```diff
@@ -55,8 +56,11 @@ const SelectChannel = ({
 		});
 
 	return (
-		<>
-			<Text style={[styles.label, { color: colors.fontTitlesLabels }]}>{I18n.t('Parent_channel_or_group')}</Text>
+		<View accessibilityLabel={`${I18n.t('Parent_channel_or_group')} - ${required ? I18n.t('Required') : ''}`}>
```

Comment:
Here you're using `-` to separate strings, but you're using `,` on SwitchItem above.
We should use always the same technique everywhere.
I'd say `,` is better.

At the same time, is it necessary to have a separator like that?
Is it going to be spoken by the screen reader?
Is it going to be faster without a separator?

Example:
Is it going to be spoken `"Label comma required"?` or is it going to give it a pause?

Reply:
```
Here you're using - to separate strings, but you're using , on SwitchItem above.
We should use always the same technique everywhere.
I'd say , is better.
```
Got it, I will extend this conversation to open.

At the same time, is it necessary to have a separator like that? on iOS it works without a separator, but on Android we need it.
Is it going to be spoken by the screen reader? No, on both platforms.
Is it going to be faster without a separator? yes, but only on android.

Reply:
we defined that accessibility separator will be ','.